libs/feature/map/src/lib/map-context/map-context.service.ts

```diff
@@ -37,7 +36,7 @@ export class MapContextService {
       mapContext = this.mergeMapConfigWithContext(mapContext, mapConfig)
     }
     if (mapContext.view) {
-      map.setView(this.createView(mapContext.view))
+      map.setView(this.createView(mapContext, map))
```

Comment:
this change should have told you that the extent was not in the correct model :)